src/PurgeClient/LocalPurgeClient.php

```diff
@@ -55,6 +55,8 @@ function ($tag) {
      */
     public function purgeAll()
     {
-        $this->cacheStore->purgeAllContent();
+        $purgeRequest = Request::create('http://localhost/', 'PURGE');
+        $purgeRequest->headers->set('X-Location-Id', '*');
```

Comment:
Didn't we have constant for `X-Location-Id` header name? I vaguely remember some discussion about it in one of the other PRs.

Reply:
yeah, but it's not really relevant class to take constant imho, as it has a special purpose.